src/torManager/torManager.ts

```diff
@@ -152,6 +164,14 @@ export class Tor {
     return onionAddress
   }
 
+  public async destroyHiddenService(serviceId: string): Promise<void> {
+    try {
+      await this.torControl.sendCommand(`DEL_ONION ${serviceId}`)
+    } catch (err) {
+      console.log(err)
```

Comment:
I don't like hiding the errors. You could e.g log the error and then return `false`. This way we could handle it on higher level and add test for unhappy path (wrong/id of already killed service). Then also add proper assertion to your happy-path test.